contracts/OnChainMultiSig.cdc

```diff
@@ -1,150 +1,169 @@
                     default:
                         panic ("Payload arg type not supported")
                 }
-                s = s.concat(b);
+                s = s.concat(b)
                 i = i + 1
             }
-            return s; 
+            return s 
         }
         
         // Verify the signature and return the total weight of valid signatures, if any.
-        pub fun verifySigners (pks: [String], sigs: [[UInt8]], currentKeyList: {String: PubKeyAttr}): UFix64? {
-            assert(pks.length == sigs.length, message: "Cannot verify signatures without corresponding public keys");
+        access(all) fun verifySigners (pks: [String], sigs: [[UInt8]], currentKeyList: {String: PubKeyAttr}): UFix64? {
+            assert(pks.length == sigs.length, message: "Cannot verify signatures without corresponding public keys")
             
-            var totalAuthorizedWeight: UFix64 = 0.0;
-            var keyList = Crypto.KeyList();
+            var totalAuthorizedWeight: UFix64 = 0.0
+            var keyList = Crypto.KeyList()
             let keyListSignatures: [Crypto.KeyListSignature] = []
-            var payloadInBytes: [UInt8] = self.getSignableData();
+            var payloadInBytes: [UInt8] = self.getSignableData()
 
             // index of the public keys and signature list
-            var i = 0;
-            var keyIndex = 0;
+            var i = 0
+            var keyIndex = 0
             while (i < pks.length) {
                 // check if the public key is a registered signer
                 if (currentKeyList[pks[i]] == nil){
-                    i = i + 1;
-                    continue;
+                    i = i + 1
+                    continue
                 }
 
                 let pk = PublicKey(
                     publicKey: pks[i].decodeHex(),
                     signatureAlgorithm: SignatureAlgorithm(rawValue: currentKeyList[pks[i]]!.sigAlgo) ?? panic ("Invalid signature algo")
                 )
                 
-                let keyListSig = Crypto.KeyListSignature(keyIndex: keyIndex, signature: sigs[i]);
-                keyListSignatures.append(keyListSig);
+                let keyListSig = Crypto.KeyListSignature(keyIndex: keyIndex, signature: sigs[i])
+                keyListSignatures.append(keyListSig)
 
                 keyList.add(
                     pk, 
                     hashAlgorithm: HashAlgorithm.SHA3_256,
                     weight: currentKeyList[pks[i]]!.weight
                 )
                 totalAuthorizedWeight = totalAuthorizedWeight + currentKeyList[pks[i]]!.weight
-                i = i + 1;
-                keyIndex = keyIndex + 1;
+                i = i + 1
+                keyIndex = keyIndex + 1
             }
             
             let isValid = keyList.verify(
                 signatureSet: keyListSignatures,
                 signedData: payloadInBytes,
+                domainSeparationTag: "FLOW-V0.0-user",
```

Comment:
This is the only change of any consequence in the `OnChainMultiSig` smart contract.

I put a placeholder in here for now, but we need to know what y'all use for the `domainSeparationTag` so we can put it here.